### PR TITLE
Add documents to request-kyc action

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

**Review by vykhrystiuk — DISMISSED** [review]

**Review by dara123 — DISMISSED** [review]

File: openapi/components/schemas/KycRequestDocuments.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+description: Documents to be requested from customer.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycRequestDocuments.yaml:1]:
```suggestion
description: Documents to be requested from a customer.
```

File: openapi/components/schemas/KycRequestDocuments.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      description: Document type to be requested from customer.
+      allOf:
+        - $ref: ./KycDocument/KycDocumentTypes.yaml
+    subtypes:
+      description: Document subtype allowed for customer.
+      type: array
+      items:
+        $ref: ./KycDocument/KycDocumentSubtypes.yaml
+    maxAttempts:
+      description: The number of document upload attempts. `0` treated as unlimited.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycRequestDocuments.yaml:18]: nitpick: based on the entity name, I'm assuming this is where the max number of attempts is set

```suggestion
      description: Total number of allowed document upload attempts. Use `0` to allow unlimited upload attempts.
```

File: openapi/components/schemas/KycRequestDocuments.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  type: object
+  required:
+    - type
+  properties:
+    type:
+      description: Document type to be requested from customer.
+      allOf:
+        - $ref: ./KycDocument/KycDocumentTypes.yaml
+    subtypes:
+      description: Document subtype allowed for customer.
```
Comment (noon-dawg) [inline openapi/components/schemas/KycRequestDocuments.yaml:13]: suggestion: I'm not sure if we need to state "for the customer here"

I feel that it is already described

```suggestion
      description: Allowed document subtype.
```

**Review by noon-dawg — COMMENTED** [review]
Minor suggestions

**Comment by noon-dawg** [comment]
LGTM

**Review by adamaltman — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Pull request merged**